src/WindowsTracing/Tracer.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "WindowsTracing/Tracer.h"
+
+#include "KrabsTracer.h"
+#include "OrbitBase/Logging.h"
+
+namespace orbit_windows_tracing {
+
+Tracer::Tracer(orbit_grpc_protos::CaptureOptions capture_options, TracerListener* listener)
+    : TracerInterface(std::move(capture_options), listener) {}
+
+void Tracer::Start() {
+  CHECK(tracer_impl_ == nullptr);
+  tracer_impl_ = std::make_unique<KrabsTracer>(capture_options_, listener_);
```

Comment:
If at any point there would be another TracerInterface implementation, we probably want some kind of factory and a generic way to instantiate the concrete tracer here (e.g. as template). But for now, we can keep it as it is.